src/backend/settings/__init__.py

```diff
@@ -352,3 +348,50 @@
 }
 
 REQUIRED_SETTINGS = ["FRONTEND_URL", "DOMAIN"]
+
+EMAIL_ADDRESS = os.getenv("EMAIL_ADDRESS", "noreply@ractf.co.uk")
+EMAIL_NAME = os.getenv("EMAIL_NAME", "RACTF")
+FROM_EMAIL = f"{EMAIL_NAME} <{EMAIL_ADDRESS}>"
+DEFAULT_FROM_EMAIL = FROM_EMAIL
+SERVER_EMAIL = FROM_EMAIL
+EMAIL_HOST = os.getenv("EMAIL_SERVER")
+EMAIL_PORT = os.getenv("EMAIL_PORT")
+EMAIL_HOST_USER = os.getenv("EMAIL_USER")
+EMAIL_HOST_PASSWORD = os.getenv("EMAIL_PASS")
+EMAIL_USE_SSL = os.getenv("EMAIL_SSL")
+EMAIL_USE_TLS = os.getenv("EMAIL_TLS")
+
+EMAIL_BACKEND = {
+    "SMTP": "django.core.mail.backends.smtp.EmailBackend",
+    "AWS": "anymail.backends.amazon_ses.EmailBackend",
+    "MAILGUN": "anymail.backends.mailgun.EmailBackend",
+    "SENDGRID": "anymail.backends.sendgrid.EmailBackend",
+    "CONSOLE": "anymail.backends.console.EmailBackend",
+    "MAILJET": "anymail.backends.mailjet.EmailBackend",
+    "MANDRILL": "anymail.backends.mandrill.EmailBackend",
+    "POSTAL": "anymail.backends.postal.EmailBackend",
+    "POSTMARK": "anymail.backends.postmark.EmailBackend",
+    "SENDINBLUE": "anymail.backends.sendinblue.EmailBackend",
+    "SPARKPOST": "anymail.backends.sparkpost.EmailBackend",
+    "TEST": "anymail.backends.test.EmailBackend",
+    "DISABLED": "django.core.mail.backends.dummy.EmailBackend"
+}[os.getenv("EMAIL_PROVIDER", "DISABLED")]
```

Comment:
I'm still not the biggest fan of this. Random arbitrary strings corresponding to python paths feels... off. Perhaps as a compromise, the backend could interpret the EMAIL_PROVIDER env-var as a path to a package, if it isn't already found in the dictionary (which isn't currently handled)

Reply:
Whilst I can see the argument for what you're suggesting, directly injecting a string into a class path feels really messy. Additionally, this implementation provides backwards compatibility with previously supported mail providers, whereas requiring the class name would be a breaking change

Reply:
I don't particularly like this solution, but I like the idea of us needing the users to dig through anymail docs to find the email backend less, users shouldnt have to care what library we use for email.

Reply:
Agreed, and I think this is starting to show the limitations of doing all config with environment variables. That might be a design decision we want to revisit at some point.